apps/miniapp/index.js

```diff
@@ -154,6 +164,44 @@ class CondoOIDCMiddleware {
     }
 }
 
+async function makeGatewaySchema () {
+    const condoExecutor = makeRemoteExecutor(CONDO_URL)
+    const appExecutor = makeRemoteExecutor(APP_URL)
+
+    return stitchSchemas({
+        subschemas: [
+            {
+                schema: await loadSchema('./schema.graphql', {
+                    loaders: [new GraphQLFileLoader()],
+                }),
+                executor: appExecutor,
+                transforms: [
+                    new RenameTypes((name) => `App${name}`),
+                    new RenameRootFields((op, name) => `app${name.charAt(0).toUpperCase()}${name.slice(1)}`),
+                ],
+            },
+            {
+                schema: await introspectSchema(condoExecutor),
```

Comment:
Probably it’s better use own hand written schema. Rather than introspected schema.